backend/pretix/__init__.py

```diff
@@ -347,3 +348,34 @@ def create_order(conference: Conference, order_data: CreateOrderInput) -> Order:
     data = response.json()
 
     return Order(code=data["code"], payment_url=data["payments"][0]["payment_url"])
+
```

Comment:
can we move this logic out of `__init__.py` file sooner or later? As I don’t expect to find anything inside usually and it’s the last place I’d look...

Reply:
in the future the pretix API will live in pythonit-toolkit